popup.js

```diff
@@ -74,13 +73,13 @@ chrome.tabs.query({active: true, currentWindow: true}, function(tabs) {
                             "Collapse": {
                                 "label": "Collapse Diff",
                                 "action": function (obj) {
-                                    port.postMessage({collapse: '^' + $node.id});
```

Comment:
Because the ids to collapse is sent as a regex, the ^ was to make sure that it was collapsing diffs that started with that node id, not diffs that may have contained that node id. However, it might have been very defensive coding, because I really can't remember what the node id looks like. Possibly, if it was just a file path, collapsing something like /lib might have side effects of collapsing subfolders also named /lib.